.github/workflows/link_check.yml

```diff
@@ -55,7 +55,7 @@ jobs:
         run: |
           if [ "${{ github.event_name }}" == "schedule" ] || [ "${{ github.event_name }}" == "workflow_dispatch" ] || ([ "${{ github.event_name }}" == "push" ] && [ "${{ github.ref }}" == "refs/heads/main" ]); then
             echo "Running link check on all notebooks in examples directory..."
-            poetry run pytest -v --check-links-ignore "https://(api|web)\.smith\.langchain\.com/.*" --check-links-ignore "https://x.com/.*" --check-links examples
+            poetry run pytest -v --check-links-ignore "https://(api|web)\.smith\.langchain\.com/.*" --check-links-ignore "https://x.com/.*" --check-links-ignore "https://docs.smith.langchain.com" --check-links examples
```

Comment:
@isahers1 you can just add docs to regex i think

Reply:
i.e. `(api|web|docs)`

Reply:
ah yeah I'm silly